nginx/Dockerfile

```diff
@@ -5,3 +5,8 @@ ADD nginx.conf /etc/nginx/nginx.conf
 # Add site
 RUN mkdir /etc/nginx/sites-enabled/
 ADD sites-enabled/ /etc/nginx/sites-enabled
+
+RUN mkdir -p /etc/ssl/private
+ADD ssl/fullchain.pem /etc/ssl/private/fullchain.pem
```

Comment:
Aren't these created directly in the sever under `/etc/ssl/` ? Why do we need to copy them from the project directory?
If we want to have them in the project under `nginx/ssl` then can we pls push that folder with a `.keep` inside?